Calliope/URLManager.swift

```diff
@@ -12,7 +12,7 @@ class URLManager {
     
     private static let calliopeDefault = "https://miniedit.calliope.cc"
     private static let makeCodeDefault = "https://makecode.calliope.cc/"
-    private static let robertaDefault = "https://lab.open-roberta.org/#loadSystem&&calliope2017​"
+    private static let robertaDefault = "https://lab.open-roberta.org/#loadSystem&&calliope2017"
```

Comment:
??? works now for me though

Reply:
I don't know. Im confused. 🤣

Reply:
> ok that was fast, thanks
> 
> i just noticed that the link does not work though, i think the year should be 2016 at least then it works.
>
> so https://lab.open-roberta.org/#loadSystem&&calliope2016 @joernalraun can you confirm this?
> 
> (so i guess 2016 is the v.1.3 revision and the other one is the 1.0 one?)

2017 is the latest one – and the one we all want. ;)

Reply:
Seems to run well. I'll merge the pull request right now and check everything for the coming update. @ichbinbenni please send me a mail if you like – as I have no time to take care of this right now, but have some ideas for some rather quick enhancements you might find useful when using the app.